src/view/components/formikReactSpectrum3/radioGroupWithDataElement.jsx

```diff
@@ -141,15 +140,16 @@ const RadioGroupWithDataElement = ({
               error={dataElementTouched && error ? error : undefined}
             >
               <ReactTextField
-                aria-label="Data Element"
+                label="Data Element"
                 value={dataElementText}
                 data-test-id={`${dataTestIdPrefix}DataElementField`}
                 onChange={newValue => setValues("dataElement", newValue)}
                 onBlur={() => dataElementSetTouched(true)}
                 validationState={
                   dataElementTouched && error ? "invalid" : undefined
                 }
-                width={width}
+                width="size-5000"
+                isRequired
```

Comment:
When the "data element" radio is selected, this textfield becomes required, so I think it makes sense to mark the textfield as required. The required indicator only shows up if we have a label though, which is why I switched the prop from `aria-label` to `label`. I added labels to other textfields that didn't have labels to be consistent.